README.md

```diff
@@ -34,7 +34,7 @@ In the example of click-through rate estimates, we first give the Google's Wide
 
 Text classification is one of the most basic tasks in natural language processing. The deep learning method can eliminate the complex feature engineering, and use the original text as input to optimize the classification accuracy.
 
-For text classification, we provide a non-sequential text classification model based on DNN and CNN. (For LSTM-based model, please refer to PaddleBook [Sentiment Analysis](https://github.com/PaddlePaddle/book/blob/develop/06.understand_sentiment/README.cn.md)).
+For text classification, we provide a non-sequential text classification model based on DNN and CNN. (For LSTM-based model, please refer to PaddleBook [Sentiment Analysis](https://github.com/PaddlePaddle/book/blob/develop/06.understand_sentiment/README.md)).
```

Comment:
How about change the link to http://www.paddlepaddle.org/docs/develop/book/06.understand_sentiment/index.html ? 
where the format of latex formula is correctly shown.
(Maybe should change the README.cn.md correspondingly)

Reply:
Good point! Done for both README.md and README.cn.md.